test/another-command/test.c

```diff
@@ -0,0 +1,13 @@
+#include "test.h"
+
+/**
+ * super-duper fast summing of values
+ */
+int64_t fast_sum(int64_t *values, int64_t n) {
+  int64_t sum = 0;
```

Comment:
This actually took me a while cause i used int first. a go int is 64bit, a c int 32bit on my machine - but obviously noone complained :/